src/modules/Dropdown/Dropdown.js

```diff
@@ -184,21 +188,19 @@ export default class Dropdown extends Component {
 
   componentWillReceiveProps(nextProps) {
     super.componentWillReceiveProps(nextProps)
-    // console.debug('Dropdown.componentWillReceiveProps()')
-    // console.log('changed props:', _.transform(nextProps, (res, val, key) => {
-    //   if (val !== this.props[key]) res[key] = val
-    // }, {}))
+    debug.groupCollapsed('componentDidUpdate()')
+    debug('changed props:', objectDiff(nextProps, this.props))
+    debug.groupEnd()
 
     if (nextProps.value !== this.state.value) {
       this.setValue(nextProps.value)
     }
   }
 
   componentDidUpdate(prevProps, prevState) { // eslint-disable-line complexity
-    // console.debug('Dropdown.componentDidUpdate()')
-    // console.log('changed state:', _.transform(this.state, (res, val, key) => {
-    //   if (val !== prevState[key]) res[key] = val
-    // }, {}))
+    debug.groupCollapsed('componentDidUpdate()')
```

Comment:
Seeing all of these methods that debug their own name, it might be nice to incorporate it as a decorator or something on the method or the class so they all do this automagically. Not a blocker, future idea.